src/Symfony/Component/Form/FormConfigBuilder.php

```diff
@@ -767,15 +767,17 @@ public function getFormConfig()
     /**
      * Validates whether the given variable is a valid form name.
      *
-     * @param string|int|null $name The tested form name
+     * @param string|null $name The tested form name
      *
      * @throws UnexpectedTypeException  if the name is not a string or an integer
      * @throws InvalidArgumentException if the name contains invalid characters
+     *
+     * @internal since Symfony 4.4
```

Comment:
This is only used in this single class. So wouldn't need to be public anymore. The whole method is not needed anyonce once we added real types in Form parameters in SF 5.